jekyll/_cci2/about-circleci.md

```diff
@@ -11,74 +10,83 @@ version:
+## Benefits of CircleCI
+{: #benefits-of-circleci }
 
-**Continuous Integration** is a key step to digital transformation.
+CircleCI runs nearly one million jobs per day in support of 30,000 organizations. Organizations choose CircleCI because jobs run fast and builds can be optimized for speed.
 
-**What?**  
-Every developer commits daily to a shared mainline.  
-Every commit triggers an automated build and test.  
-If build and test fails, it’s repaired quickly - within minutes.
+CircleCI can be configured to run very complex pipelines efficiently with sophisticated [caching,]({{site.baseurl}}/2.0/caching/) [docker layer caching,]({{site.baseurl}}/2.0/docker-layer-caching/) and [resource classes]({{site.baseurl}}/2.0/optimizations/#resource-class) for running on faster machines.
 
-**Why?**  
-Improve team productivity, efficiency, happiness.  
-Find problems and solve them, quickly.  
-Release higher quality, more stable products.
+As a developer using CircleCI you can:
+- [SSH into any job]({{site.baseurl}}/2.0/ssh-access-jobs/) to debug your build issues
+- Set up [parallelism]({{site.baseurl}}/2.0/parallelism-faster-jobs/) in your [.circleci/config.yml]({{site.baseurl}}/2.0/configuration-reference/) file to run jobs faster
+- Configure [caching]({{site.baseurl}}/2.0/caching/) with two simple keys to reuse data from previous jobs in your [workflow]({{site.baseurl}}/2.0/workflows/)
+- Configure self-hosted [runners]({{site.baseurl}}/2.0/runner-overview/) for unique platform support
+- Access [Arm resources]({{site.baseurl}}/2.0/arm-resources/) for the machine executor
+- Use [orbs]({{site.baseurl}}/2.0/using-orbs/), reusable packages of configuration, to integrate with third parties
+- Use pre-built Docker [images]({{site.baseurl}}/2.0/circleci-images/) in a variety of languages
+- Use the [API](https://circleci.com/docs/api/v2/) to retreive information about jobs and workflows
+- Use the [CLI]({{site.baseurl}}/2.0/local-cli/) to access advanced tools locally
+- Get flaky test detection with [test insights]({{site.baseurl}}/2.0/insights-tests/)
 
+As an operator or administrator of CircleCI installed on your own servers, CircleCI provides monitoring and insights into your builds and uses [Nomad](https://www.nomadproject.io/) for scheduling.
 
-## Free trial options
-{: #free-trial-options }
+See the [CircleCI Operations Guide]({{site.baseurl}}/2.0/server-3-overview/) for complete server documentation.
 
-CircleCI provides a free trial with the following options:
+## Pricing and trial options
+{: #pricing-and-trial-options }
 
-- **Cloud**: See [Signup and Try CircleCI]({{site.baseurl}}/2.0/first-steps/) to get started with the hosted application.
+Visit CircleCI's [Pricing page](https://circleci.com/pricing/) to view free and paid options. 
 
-### Open source
-{: #open-source }
+To get started with a free trial, visit the [Sign Up and Try CircleCI]({{site.baseurl}}/2.0/first-steps/) page and get set up with the hosted cloud application.
```

Comment:
Rather than linking to this doc I think just link to the signup page, this doc might get repurposed or archived soon I think